engine/table/src/main/java/io/deephaven/engine/table/impl/replay/Replayer.java

```diff
@@ -60,8 +62,9 @@ public Replayer(Instant startTime, Instant endTime) {
     @Override
     public void start() {
         deltaNanos = DateTimeUtils.millisToNanos(System.currentTimeMillis()) - DateTimeUtils.epochNanos(startTime);
-        for (Runnable currentTable : currentTables) {
-            updateGraph.addSource(currentTable);
+        updateGraph.addSource(sourceRefresher);
+        for (ReplayTableBase currentTable : currentTables) {
+            currentTable.start();
         }
```

Comment:
It might be worth a forward-looking comment that points out that the ReplayTables are allowed to run in parallel with the Replayer, and that maybe we might want an UpdateSourceCombiner in the future.